src/app/(web)/page.tsx

```
@@ -146,8 +113,25 @@ const results = [
+    potentialAction: {
+      '@type': 'SearchAction',
+      target: `${siteConfig.url}/tournaments`,
+      'query-input': 'required name=search_term_string',
+    },
+  };
+
   return (
     <>
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
```

Biome (2.4.7)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)